### DISCUSSION TITLE
Episode 168: Parallel Mode

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#60] [Episode]
Episode Link: https://adspthepodcast.com/2024/02/09/Episode-168.html

Feel free to write comments or ask questions 🙂

**Comment by BraxtonMDevelops** [comment]
This has been the best podcast i've heard in a while. Never expected such innovations as bringing back "showing" Bryce code doing a live covid test and lego going on in teh background.

**Comment by siemato** [comment]
For contacting Andrew, i recommend talking to Loris  (@kristoff-it), Zig's VP of Community. I passed the message along to him and he's waiting for a message from you.

**Reply by codereport** [reply]
I just DM-ed Loris on twitter. Thanks for the tip : )

**Comment by kentf** [comment]
2 things.

The audio on Spotify is getting out of sync around the 11 minute mark. You talk at the same time, and there is silence at the same time. It's somewhat difficult to follow, and a little creepy 😁, 

The link to GitHub discussion for ep 168 points to episode 167

Cheers! And get well soon.

**Reply by codereport** [reply]
Yea, I sent a [Tweet](https://x.com/adspthepodcast/status/1755979792174768469?s=20) out about the audio sync issue. It was broken for the first 30-60 minutes and I uploaded a fixed version but you would have needed to redownloaded it to get the fixed one. And thanks for the link tip. Fixed now as well!

**Reply by kentf** [reply]
The one time the Spotify client was quick about downloading an episode after a release... 😅

**Comment by roibarkan** [comment]
Wanted to mention that on my cppcon talk "more ranges please" I [suggested](https://youtu.be/Ugb8-ig7Pnw?t=47m23s) making RLE (run length encoding) a part of the ranges library - and called it histogram.
Thought the ADSP audience might like some of that talk
